tensorboard/data/provider.py

```diff
@@ -388,8 +388,8 @@ def list_hyperparameters(self, ctx=None, *, experiment_ids):
             experiments.
 
         Returns:
-          A Collection[Hyperparameter] describing the hyperparameter metadata
-          for the experiments.
+          A ListHyperparametersResult describing the hyperparameter-related
```

Comment:
This function should now return en empty `ListHyperparametersResult` instead of `[]`.

Reply:
Nice catch. Done.